app/Models/Project.php

```diff
@@ -17,6 +17,7 @@
  * @property string $template
  * @property bool $published
  * @property string $color
+ * @property int $sort_order
```

Comment:
```suggestion
 * @property ?int $sort_order
```

Reply:
Or maybe it should have been defaulted by some number (like zero?) in database migration?

Reply:
Added nullable symbol, I think that it's ok to have it nullable, there is no problem with applying order.